File: JAVA/Day2/TestTask3.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class TestTask3 {

    @Test
    public void testNoOneLikesThis() {
        String result = Task3.getLikes(new String[]{});
        assertEquals("no one likes this", result);
    }

    @Test
    public void testSinglePersonLikesThis() {
        String result = Task3.getLikes(new String[]{"Peter"});
        assertEquals("Peter likes this", result);
    }

    @Test
    public void testTwoPeopleLikeThis() {
        String result = Task3.getLikes(new String[]{"Jacob", "Alex"});
        assertEquals("Jacob and Alex like this", result);
    }

    @Test
    public void testMoreThanThreePeopleLikeThis() {
        String result = Task3.getLikes(new String[]{"Alex", "Jacob", "Mark", "Max"});
        assertEquals("Alex, Jacob and 2 others like this", result);
    }
}

File: Python/Day2/TestTask3.py
import unittest
import Task3 

class TestTask3(unittest.TestCase):
    def test_no_one_likes_this(self):
        result = Task3.get_likes([])
        self.assertEqual(result, "no one likes this")

    def test_single_person_likes_this(self):
        result = Task3.get_likes(["Peter"])
        self.assertEqual(result, "Peter likes this")

    def test_two_people_like_this(self):
        result = Task3.get_likes(["Jacob", "Alex"])
        self.assertEqual(result, "Jacob and Alex like this")

    def test_more_than_three_people_like_this(self):
        result = Task3.get_likes(["Alex", "Jacob", "Mark", "Max"])
        self.assertEqual(result, "Alex, Jacob and 2 others like this")